### PR TITLE
Add 228 json html themes for to html

### PR DESCRIPTION
**Pull request by fdncred**
pretty big change here
- removed old assets, added new zipped asset
- added --list to get a list of the theme names
- reworked some older theme code
- added rust-embed and zip crate
- removed the dark tests

### CODE REVIEW

File: crates/nu-cli/src/commands/to_html.rs [hunk 1]
```diff
@@ -6,7 +6,104 @@ use nu_errors::ShellError;
...
+
+impl Default for HtmlThemes {
+    fn default() -> Self {
+        HtmlThemes {
+            themes: vec![HtmlTheme::default()],
+        }
+    }
+}
+
+// impl Default for HtmlTheme {
```
Comment (jonathandturner) [inline crates/nu-cli/src/commands/to_html.rs:53]: You can just delete this. If we want to update the theme, should be easy enough

File: crates/nu-cli/src/commands/to_html.rs [hunk 2]
```diff
@@ -60,183 +159,135 @@ impl WholeStreamCommand for ToHTML {
...
-        hm.insert("cyan", "#037979".to_string());
-        hm.insert("white", "white".to_string());
-
-        hm.insert("background", "white".to_string());
-        hm.insert("foreground", "black".to_string());
+    if th.name.to_lowercase().eq(&"campbell".to_string()) // this just means no theme was passed in
+        && theme.is_some()
+    // this means there was a theme passed in
+    {
+        println!(
```
Comment (jonathandturner) [inline crates/nu-cli/src/commands/to_html.rs:199]: This should probably be a ShellError, I think?

File: crates/nu-cli/src/commands/to_html.rs (hunk 1 above)
Comment (fdncred) [reply crates/nu-cli/src/commands/to_html.rs:53]: oh, ya. that was a last minute thing. i was defaulting to campbell but then it made every test fail, so i commented it out to test the original and forgot to delete it. i'll remove it in the morning. thx.

File: crates/nu-cli/src/commands/to_html.rs (hunk 2 above)
Comment (fdncred) [reply crates/nu-cli/src/commands/to_html.rs:199]: ok. I may need help with that. I'll dig around in the source to see if I can find an example of what that looks like.

Comment (fdncred) [reply crates/nu-cli/src/commands/to_html.rs:199]: Now that I think about this, I was printing a warning here, notifying the user that their theme name couldn't be found and that I was using the default theme. If I use ShellError, it appears that I have to fail versus give a warning. I'm not sure which we prefer to do. Any suggestions?

Comment (jonathandturner) [reply crates/nu-cli/src/commands/to_html.rs:199]: We probably generally want some guidelines for when we give a warning vs an error. For this case, I think it'd be okay to error for now. In the future, we could do a pass over where we error and see if some of those could become warnings.

**Comment by jonathandturner** [comment]
Looks good

**Pull request merged**